data/json/items/gun/223.json

```diff
@@ -146,6 +146,40 @@
     ],
     "magazines": [ [ "223", [ "famasmag" ] ] ]
   },
+  {
+    "id": "fs2000",
+    "copy-from": "rifle_semi",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": "FS2000",
+    "description": "A sleek bullpup carbine designed by FN Herstal, complete with an integrated sight accessory rail.  The forward ejecting action and ambidextrous controls make firing comfortable for both left and right-handed shooting.  The whole rifle is well sealed from mud and dust for reliability, but this makes it incompatible with many aftermarket magazines.",
+    "weight": "3319 g",
+    "volume": "7986 ml",
+    "price": 125000,
+    "to_hit": -1,
+    "bashing": 12,
+    "material": [ "steel", "plastic" ],
+    "symbol": "(",
+    "color": "dark_gray",
+    "ammo": "223",
+    "ranged_damage": { "damage_type": "stab", "amount": -2 },
+    "dispersion": 150,
+    "durability": 9,
+    "min_cycle_recoil": 1350,
+    "default_mods": [ "factory_handguard" ],
+    "magazines": [ [ "223", [ "stanag30" ] ] ],
+    "magazine_well": "228 ml",
```

Comment:
For good or ill, magazine wells were removed by the container overhaul so this is causing a load error.
```suggestion
```